k6.js

```diff
@@ -0,0 +1,26 @@
+import http from 'k6/http';
+import { check } from 'k6';
+
+export let options = {
+  stages: [
+    { duration: '30s', target: 10 },
+    { duration: '30s', target: 20 },
+    { duration: '30s', target: 40 },
+    { duration: '30s', target: 80 },
+    { duration: '30s', target: 160 },
+    { duration: '30s', target: 0 }
+  ]
+};
+
+export default function () {
+  let listingId = Math.floor((Math.random() * 10000000));
```

Comment:
I believe this could technically produce 0 as a listing id, I think you need to add a + 1 after (Math.random() * 10000000)

Reply:
or use Math.ceil as well